src/attributes/attr.js

```diff
@@ -38,7 +38,8 @@ jQuery.extend({
 
 		// All attributes are lowercase
 		// Grab necessary hook if one is defined
-		if ( nType !== 1 || !jQuery.isXMLDoc( elem ) ) {
+		notxml = nType !== 1 || !jQuery.isXMLDoc( elem );
+		if ( notxml ) {
```

Comment:
This variable might not be benefiting us in either method. I'd check the impact of removing it.